hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This adds a new minimizer step, which reorders byte arrays: 3 1 2 becomes 1 2 3.
+The test suite continues to pass; this is believed to be backwards-compatible.
```

Comment:
This line probably isn't needed - the test suite always passes for releases, and it's definitely backwards compatible! (The shrunk result is not part of the public API)